scripts/lint.sh

```diff
@@ -1,34 +1,38 @@
 #!/usr/bin/env bash
-set -euxo pipefail
+set -euo pipefail
 
-# Check Go Formatting
+echo -n "Formatting golang code... "
 gofmtout=$(go fmt ./...)
-test -z "${gofmtout}"
+if [ "$gofmtout" ]; then
+  printf "\n\n"
+  echo "Files with formatting errors:"
+  echo "${gofmtout}"
+  exit 1
+fi
+echo "done."
 
-# Run Golint
+echo -n "Linting golang code... "
 # TODO: Make sure golangci-lint is installed and ready to be run
 GOLANG_LINT_CMD=golangci-lint
 
-if [ ! command -v ${GOLANG_LINT_CMD} ]; then
-  echo "$GOLANG_LINT_CMD not found, fetching..."
+if [ ! "$(command -v ${GOLANG_LINT_CMD})" ]; then
+  echo -n "installing ${GOLANG_LINT_CMD}... "
   go get -u github.com/golangci/golangci-lint/cmd/golangci-lint
 fi
 
-golangci-lint run --disable-all -E golint ./pkg/...
-golangci-lint run --disable-all -E golint ./cmd/...
-golangci-lint run --disable-all -E golint ./tests/...
+golangci-lint run \
+  --exclude "composite literal uses unkeyed fields" \
```

Comment:
Here's an example of where we'd hit this:

```golang
type MyCustomError struct {
  message string
}
func (e *MyCustomError) Error() string {
  return message
}

// This would cause a linting error,
err := &MyCustomError{ "some error message" }

// Linter wanted us to do:
&MyCustomError{ message: "some error message" }

// but I think that's more verbose that necessary
```